file.go

```diff
@@ -125,10 +125,15 @@ func AtomicWriteFileAndChange(filename string, contents []byte, change func(*os.
 	if _, err := f.Write(contents); err != nil {
 		return fmt.Errorf("cannot write %q contents: %v", filename, err)
 	}
+	if err := f.Sync(); err != nil {
+		return err
+	}
+	if err := f.Close(); err != nil {
+		return err
+	}
```

Comment:
In that case though the interface should really be changed - the change func should be passed the file name instead of a closed `*File`. There's only one use of `AtomicWriteFileAndChange` in the whole codebase, so it's probably ok to update it.